### ISSUE TITLE
Deleting a domain doesn't update email addresses. Update them leads to a blocking exception.

### ISSUE THREAD

**Issue by e-lie** [YunoHost/issues#1282]
Yunohost version 3.3.4 and admin 3.3.1, moulinette 3.3.1

After importing users, removing the domain of their addresses doesn't block or warn.

Trying to update the email addresses (even when the corresponding domain has been added back) leads to : 
### Action
```
PUT /users/

{"username":"elie","firstname":"Elie","locale":"fr","lastname":"Gavoty","mail":<updated email>","remove_mailalias":["<old aliases>"]}
```
### Trace
```
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/moulinette/interfaces/api.py", line 405, in process
    ret = self.actionsmap.process(arguments, timeout=30, route=_route)
  File "/usr/lib/python2.7/dist-packages/moulinette/actionsmap.py", line 519, in process
    return func(**arguments)
  File "/usr/lib/moulinette/yunohost/log.py", line 276, in func_wrapper
    result = func(*args, **kwargs)
  File "/usr/lib/moulinette/yunohost/user.py", line 321, in user_update
    main_domain = _get_maindomain()
NameError: global name '_get_maindomain' is not defined
```

**Comment by alexAubin** [comment]
This got fixed in 3.4.x (the import was missing) : https://github.com/YunoHost/yunohost/blob/stretch-testing/src/yunohost/user.py#L284

Can you try to patch stuff locally and see if everything behave as expected ?

**Comment by e-lie** [comment]
Okay the patch fixes it. thx !
